Assignment1/test/test_distance_formula.py

```diff
@@ -1,11 +1,92 @@
 import pytest
+from pytest import approx
```

Comment:
If you need approx, just use the pytest import and call pytest.approx
You might have to wrap a couple lines to pass pep8, but it will make it more apparent that `approx` is part of the testing framework and not the source code.